doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -20,7 +20,7 @@ Interactive use (coqtop)
 ------------------------
 
 In the interactive mode, also known as the Coq toplevel, the user can
```

Comment:
```suggestion
In the interactive mode, also known as the Coq toplevel, the user can
```

Reply:
I've gone ahead and updated each instance that @Alizter changed to be plural and updated the title of the PR to better reflect the multiple changes.